integration-tests/jpa-postgresql/pom.xml

```diff
@@ -169,7 +174,7 @@
                         <configuration>
                             <images>
                                 <image>
-                                    <name>postgres:10.5</name>
+                                    <name>${postgresql.image}</name>
```

Comment:
It should be consistent with `postgres.url` a few lines above. So either we fix the existing one (and the CI jobs) or we stick to `postgres` as the prefix.

Reply:
Updated and pulled up the url property so all 4 modules are in sync